File: projects/password-generator/csharp/Gen.cs
using System;
using System.Text;

class Gen {
    static void Main() {
        const string chars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!@#$%";
        int length = 12;
        var rand = new Random();
        var password = new StringBuilder();

        for (int i = 0; i < length; i++) {
            password.Append(chars[rand.Next(chars.Length)]);
        }
        Console.WriteLine("Password: " + password);
    }
}

File: projects/password-generator/java/Gen.java
import java.util.Random;

public class Gen {
    public static void main(String[] args) {
        String chars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!@#$%";
        int length = 12;
        Random rand = new Random();
        StringBuilder password = new StringBuilder();

        for (int i = 0; i < length; i++) {
            password.append(chars.charAt(rand.nextInt(chars.length())));
        }
        System.out.println("Password: " + password);
    }
}
